js/main.js

```diff
@@ -0,0 +1,32 @@
+const navToggle = document.querySelector('.nav-toggle');
+const navbar = document.querySelector('.mobile-nav-hidden');
+const closeHamburger = document.querySelector('.hidden');
+const hamburger = document.querySelector('#hamburger');
+const closeMenu = document.querySelector('.mobile-nav-menu');
+const navIcon = document.querySelectorAll('.navIcon');
+
+hamburger.addEventListener('click', () => {
+  navbar.classList.add('header');
```

Comment:
- [Optional] Please try to change the text color of the `language and framework` to `background: #9490FF;` as it is in the Figma design, currently, yours is `#000`


##### Your design

![Screenshot from 2022-06-17 12-39-23](https://user-images.githubusercontent.com/58771507/174291427-2379eff0-0b2e-49ff-af6a-a1cf6513ea61.png)

##### Figma design 
![Screenshot from 2022-06-17 12-36-02](https://user-images.githubusercontent.com/58771507/174291359-5c27b433-2f5b-47a8-8b29-3dc257f517b3.png)

Reply:
Thanks, I couldn't see this because I had conflicts. Changes have been effected